MMDevice/MMDevice-SharedRuntime.vcxproj

```diff
@@ -35,14 +35,14 @@
     <ProjectGuid>{B8C95F39-54BF-40A9-807B-598DF2821D55}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>MMDeviceSharedRuntime</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.19041.0</WindowsTargetPlatformVersion>

```

Comment:
Is editing this file necessary?

Reply:
I just followed one of their latest vcxproj files

Reply:
My issue is that it’s something outside of our ‘addition’ … it would be good if we could avoid changing things outside our folder